recipes/paragraph/LICENSE.TXT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
Looks like the License file is packaged in the tarball, so please remove this file.